ui/src/modules/app_list/index.tsx

```diff
@@ -305,7 +305,7 @@ const AppCard: React.FC<{ app: ApplicationInfo }> = ({ app }) => {
 
         <Stack gap={0}>
           <Text c="gray.6" fz="sm" truncate>
-            Owned by Default User
+            Owned by {app.user || 'Default User'}
```

Comment:
Use "unknown" instead default user. Because the user field should not be empty by design. If it's empty, it's a bug.

Reply:
Done.